### PR TITLE
Calculate and Report FOV from Calibration Coefficients

### PR DESCRIPTION
**Pull request by AnotherUser12**
Non-invasive addition to display empirically calculated horizontal, vertical, and diagonal, FOVs. Calculations are simple, summarized here: https://stackoverflow.com/questions/31257664/relation-between-horizontal-vertical-and-diagonal-field-of-view

Btw, principal point is assumed to be on-center.

### CODE REVIEW

**Comment by mcm001** [comment]
Can you undo package lock changes? Otherwise lgtm

File: photon-server/src/main/resources/web/index.html [hunk 1]
```diff
@@ -1 +1 @@
-<p>UI has not been copied!</p>
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/favicon.png><title>PhotonVision</title><link href=/js/chunk-2d216214.4302abb0.js rel=prefetch><link href=/js/chunk-2d216257.0de4b8cc.js rel=prefetch><link href=/js/chunk-6bdce92c.119701f5.js rel=prefetch><link href=/css/app.b5a35c37.css rel=preload as=style><link href=/css/chunk-vendors.db45c662.css rel=preload as=style><link href=/js/app.1a8eed75.js rel=preload as=script><link href=/js/chunk-vendors.577f2286.js rel=preload as=script><link href=/css/chunk-vendors.db45c662.css rel=stylesheet><link href=/css/app.b5a35c37.css rel=stylesheet></head><body><noscript><strong>We're sorry but PhotonVision doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.577f2286.js></script><script src=/js/app.1a8eed75.js></script></body></html>
```
Comment (mcm001) [inline photon-server/src/main/resources/web/index.html:1]: Revert these changes as well plz

File: photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java [hunk 2]
```diff
@@ -538,6 +538,8 @@ public PhotonConfiguration.UICameraConfiguration toUICameraConfig() {
             internalMap.put("height", c.resolution.height);
             internalMap.put("intrinsics", c.cameraIntrinsics.data);
             internalMap.put("extrinsics", c.cameraExtrinsics.data);
+            System.out.println(c.cameraIntrinsics.data);
```
Comment (mcm001) [inline photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java:541]: Yeet print?

File: photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NetworkTablesManager.java [hunk 3]
```diff
@@ -73,7 +73,7 @@ public void accept(LogMessage logMessage) {
     }
 
     public void broadcastConnectedStatus() {
-        TimedTaskManager.getInstance().addOneShotTask(this::broadcastConnectedStatusImpl, 1000L);
+        TimedTaskManager.getInstance().addOneShotTask(this::broadcastConnectedStatusImpl, 1000000000000000L);
```
Comment (mcm001) [inline photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NetworkTablesManager.java:76]: Revert?

File: photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java (hunk 2 above)
Comment (AnotherUser12) [reply photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java]: lol forgot bout that

**Review by mcm001 — APPROVED** [review]
